actions/websocket_actions.jsx

```diff
@@ -412,13 +413,11 @@ function handleUserUpdatedEvent(msg) {
     const user = msg.data.user;
 
     if (currentUser.id === user.id) {
-        dispatch({
-            type: UserTypes.RECEIVED_ME,
-            data: {
-                ...currentUser,
-                last_picture_update: user.last_picture_update
-            }
-        });
+        if (user.update_at > currentUser.update_at) {
+            // Need to request me to make sure we don't override with sanitized fields from the
+            // websocket event
+            getMe()(dispatch, getState);
```

Comment:
The user sent by the WS event will have sanitized fields, but it's difficult to tell what is sanitized and what is actually blank. Easiest solution I could think of was to just request yourself again if the update_at for the WS provided user is newer. This will prevent the client making the request from making this request but other tabs/clients will make it.